clang/test/CIR/CodeGen/func_dsolocal_pie.c

```diff
@@ -0,0 +1,34 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-android21 -fclangir -emit-cir -pic-is-pie -pic-level 1 %s -o %t1.cir
+// RUN: FileCheck --input-file=%t1.cir %s -check-prefix=CIR
+// RUN: %clang_cc1 -triple aarch64-none-linux-android21 -fclangir -emit-llvm -pic-is-pie -pic-level 1 %s -o %t.ll
+// RUN: FileCheck --input-file=%t.ll %s -check-prefix=LLVM
```

Comment:
```suggestion
// RUN: %clang_cc1 -triple aarch64-none-linux-android21 -fclangir -emit-llvm -pic-is-pie -pic-level 1 %s -o %t-cir.ll
// RUN: FileCheck --input-file=%t-cir.ll %s -check-prefix=LLVM
// RUN: %clang_cc1 -triple aarch64-none-linux-android21 -emit-llvm -pic-is-pie -pic-level 1 %s -o %t.ll
// RUN: FileCheck --input-file=%t.ll %s -check-prefix=OGCG
```
This is the kind of change I was suggesting for tests that have LLVM IR checks. You would also need to add OGCG checks, which should be very close to the existing LLVM checks. The point is to find differences between the LLVM IR generated when we go through CIR compared to the LLVM IR generated when we are not using CIR.

As I mentioned before, it's probably best to remove tests that need this kind of update and make that a follow-up PR.